src/domain/event/EditEventPage.tsx

```diff
@@ -53,14 +59,57 @@ const EditEventPage: React.FC = () => {
 
   const submit = async (values: EventFormFields) => {
     try {
-      await editEvent({
-        variables: {
-          event: {
-            id: eventData?.event?.id || '',
-            ...getEventPayload(values, selectedLanguage),
+      const requests = [];
+
+      requests.push(
+        editEvent({
+          variables: {
+            event: {
+              id: eventData?.event?.id || '',
+              ...getEventPayload(values, selectedLanguage),
+            },
           },
-        },
-      });
+        })
+      );
+
+      const shouldSaveImage =
```

Comment:
this could be moved to its own functions and be called in the if statetement to make this submit function little bit cleaner